config/routes.rb

```diff
@@ -20,12 +20,16 @@
 
   get "/trainees/not-supported-route", to: "trainees/not_supported_routes#index"
 
+  get "/sign-in" => "sign_in#index"
+  get "/sign-out" => "sign_out#index"
+
   if FeatureService.enabled?("use_dfe_sign_in")
     get "/auth/dfe/callback" => "sessions#callback"
-    get "/auth/dfe/sign-out" => "sessions#redirect_after_dsi_signout"
+    get "/auth/dfe/signout" => "sessions#signout"
```

Comment:
Is this configurable. Bit annoying that we aren't consistent with the `sign-out` vs `signout`

Reply:
changed to signout